thunder/core/transform_common.py

```diff
@@ -174,7 +192,9 @@ def _helper(x):
                 needed_proxies.add(variableify(x))
 
     dcetrace = from_trace(trace)
-    dcetrace.bound_symbols = list(reversed(dced))
+    dced_bound_symbols = list(reversed(dced))
+    dced_bound_symbols = dced_bound_symbols
```

Comment:
What's up with this line?

Reply:
Ahahah. I prepared the next line and then forgot the preparations.